cves/kernel/CVE-2018-10940.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
Consider starting with a more formal introduction, like "This vulnerability involves..."